dev-python/livereload/livereload-2.6.1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1 vcs-snapshot
```

Comment:
Hmm, I don't think *vcs-snapshot* is needed here.

Reply:
I got: "Source directory ${S} doesn't exist" without  vcs-snapshot, but I fixed it by adding `S="${WORKDIR}/python-${P}"`